packages/klevu-ui/src/components/klevu-pagination/klevu-pagination.css

```diff
@@ -5,6 +5,7 @@
   gap: var(--klevu-spacing-03);
   justify-content: center;
   flex-wrap: wrap;
+  padding: var(--klevu-spacing-03) 0;
```

Comment:
padding should not be added to components itself. Only in the place they are used. This change is coming up in other MR too (if I remember right).

Reply:
added padding to pagination as suggested